src/WorkItemMigrator/JiraExport/RevisionUtils/FieldMapperUtils.cs

```diff
@@ -158,6 +160,43 @@ public static object MapSprint(string iterationPathsString)
             return iterationPath;
         }
 
+        private static readonly Dictionary<string, decimal> CalculatedLexoRanks = new Dictionary<string, decimal>();
+        private static readonly Dictionary<decimal, string> CalculatedRanks = new Dictionary<decimal, string>();
+
+        private static readonly Regex LexoRankRegex = new Regex(@"^[0-2]\|[0-9a-zA-Z]*(\:[0-9a-zA-Z]*)?$",
+            RegexOptions.Compiled | RegexOptions.CultureInvariant | RegexOptions.Singleline);
+
+        public static object MapLexoRank(string lexoRank)
+        {
+            if (string.IsNullOrEmpty(lexoRank) || !LexoRankRegex.IsMatch(lexoRank))
+                return decimal.MaxValue;
+
+            if (CalculatedLexoRanks.ContainsKey(lexoRank))
+            {
+                Logger.Log(LogLevel.Warning, "Duplicate rank detected. You may need to re-balance the JIRA LexoRank. see: https://confluence.atlassian.com/adminjiraserver/managing-lexorank-938847803.html");
+                return CalculatedLexoRanks[lexoRank];
+            }
+
+            // split by bucket and sub-rank delimiters
+            var lexoSplit = lexoRank.Split(new[] {'|', ':'}, StringSplitOptions.RemoveEmptyEntries);
+
+            // calculate the numeric value of the rank and sub-rank (if available)
+            var b36Rank = Base36.Decode(lexoSplit[1]);
+            var b36SubRank = lexoSplit.Length == 3 && !string.IsNullOrEmpty(lexoSplit[2])
+                ? Base36.Decode(lexoSplit[2])
+                : 0L;
+
+            // calculate final rank value
+            var rank = Convert.ToDecimal($"{b36Rank}.{b36SubRank}");
```

Comment:
This doesn't work for me (the tests fail) unless I change the dot to a comma to match my culture settings. Suggested fix: 
`var rank = Convert.ToDecimal($"{b36Rank}.{b36SubRank}", CultureInfo.InvariantCulture.NumberFormat);`

Reply:
Will make suggested change

Reply:
This should be updated now